node-tests/get-babel-options-test.js

```diff
@@ -164,20 +175,37 @@ describe("get-babel-options", function () {
       const config = {
         "ember-cli-babel": { enableTypeScriptTransform: false },
       };
-      let plugins = _addDecoratorPlugins(
-        ["@babel/plugin-transform-typescript"],
-        {},
+      let plugins = _addDecoratorPlugins({
+        plugins: ["@babel/plugin-transform-typescript"],
+        options: {},
         config,
-        this.addon.parent,
-        this.addon.project
-      );
+        parent: this.addon.parent,
+        project: this.addon.project,
+        isClassPropertiesRequired: true,
+      });
 
       expect(plugins.length).to.equal(3, "class fields and decorators added");
       expect(plugins[2]).to.equal(
         "@babel/plugin-transform-typescript",
         "typescript is now last"
       );
     });
+
+    it("should not include class fields if they are not required", function () {
+      let plugins = _addDecoratorPlugins({
+        plugins: [],
+        options: {},
+        config: {},
+        parent: this.addon.parent,
+        project: this.addon.project,
+        isClassPropertiesRequired: false,
+      });
+      expect(plugins.length).to.equal(1, "only one plugin");
+      expect(plugins[0][0]).to.include(
+        "@babel/plugin-proposal-decorators",
```

Comment:
This fails on windows (IMO for annoying reasons :P)

Reply:
Updated (should fix windows failures)

Reply:
Wat. The fix is bananas! TIL, though.